packages/react-native-web/src/exports/Modal/ModalFocusTrap.js

```diff
@@ -123,6 +124,19 @@ const ModalFocusTrap = ({ active, children }: ModalFocusTrapProps) => {
     }
   }, [active]);
 
+  // To be fully compliant with WCAG we need to refocus element that triggered opening modal
+  // after closing it
+  useEffect(() => {
+    if (canUseDOM) {
+      lastFocusedElementOutsideTrap.current = document.activeElement;
+      return () => {
+        if (lastFocusedElementOutsideTrap.current) {
```

Comment:
This value needs to be stored in the closure rather than read in the cleanup callback